src/com/projectkorra/projectkorra/GeneralMethods.java

```diff
@@ -903,13 +903,25 @@ public static Location getRightSide(final Location location, final double distan
 	}
 
 	public static Location getMainHandLocation(final Player player) {
-		Location loc;
+		double y = 1.2 - (player.isSneaking() ? 0.4 : 0);

```

Comment:
Is there a reason this algorithm isn't consistent with getOffHandLocation()?

L917 you subtract by .38 instead of .4.

Reply:
This was an oversight. Good spotting